src/main/java/org/folio/circulation/resources/ExpiredSessionProcessingResource.java

```diff
@@ -62,13 +64,12 @@ private void process(RoutingContext routingContext) {
   }
 
   private CompletableFuture<Result<Void>> attemptEndSession(
-    PatronActionSessionService patronSessionService, ExpiredSession expiredSession) {
+    PatronActionSessionService patronSessionService, List<ExpiredSession> expiredSessions) {
 
-    if (expiredSession == null || StringUtils.isBlank(expiredSession.getPatronId())) {
+    if (expiredSessions.isEmpty()) {
```

Comment:
Should we also check if patronId is blank and maybe filter the list?

Reply:
done